packages/terra-site/src/examples/section-header/ClosedSectionHeader.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import SectionHeaderExampleTemplate from './SectionHeaderExampleTemplate';
+
+const sectionHeaderProps = {
+  title: 'Closed',
+  onClick: () => {},
```

Comment:
Do we need to provide the onClick?

Reply:
We do in order for the accordion to appear